src/Mvc/Mvc.Core/src/Builder/OpenApiEndpointConventionBuilderExtensions.cs

```diff
@@ -27,6 +27,20 @@ public static MinimalActionEndpointConventionBuilder ExcludeFromDescription(this
             return builder;
         }
 
+        /// <summary>
+        /// Adds the <see cref="ITagsMetadata"/> to <see cref="EndpointBuilder.Metadata"/> for all builders
+        /// produced by <paramref name="builder"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="MinimalActionEndpointConventionBuilder"/>.</param>
+        /// <param name="tags">A collection of tags to be associated with the endpoint.</param>
+        /// <returns>A <see cref="MinimalActionEndpointConventionBuilder"/> that can be used to further customize the endpoint.</returns>
+
+        public static MinimalActionEndpointConventionBuilder WithTags(this MinimalActionEndpointConventionBuilder builder, params string[] tags)
```

Comment:
`MinimalActionEndpointConventionBuilder` has been renamed to `DelegateEndpointConventionBuilder`.
```suggestion
        public static DelegateEndpointConventionBuilder WithTags(this DelegateEndpointConventionBuilder builder, params string[] tags)
```